plugins/foreman_openscap/1.0/index.md

```diff
@@ -0,0 +1,630 @@
+* By assigning a policy to host(s) via hosts index: In the hosts index, choose the hosts you wish to assign a policy. Click on 'Select Action' and choose: "Assign Compliance Policy" and choose which policy to assign to the selected hosts.
+
+![Assign Hosts]({{page.images}}/assign_hosts.png)
+
+## 4.4 ARF Reports
+You can access the generated reports via Hosts -> Compliance -> Reports
+![ARF reports index]({{page.images}}/arf_index.png)
+
+Report page gives information about individual rules that were checked during scan. Here you can download the actual report generated by OpenSCAP in HTML or as XML in bzip archive.
+![View ARF report]({{page.images}}/view_arf_report.png)
+
+Clicking on "View full report" at the top of the page will lead you to the actual security audit report generated by OpenSCAP, with detailed information on the host's security check and suggested remediation.
+![ARF report]({{page.images}}/arf_report.png)
+
+## 4.5 Tailoring files ( foreman_openscap >= 0.6.5 )
+
+Using a Tailoring File effectively allows you to modify a policy. You can assign a Tailoring File to a Policy when creating / updating a policy. Because Tailoring File may contain multiple profiles, you have to select your modified profile as well.
+
+### 4.5.1 Creating a Tailoring file
+
+You can create a new Tailoring file with [SCAP Workbench](https://www.open-scap.org/tools/scap-workbench/)
+
+### 4.5.2 Uploading a Tailoring file
+
+__Access Tailoring files__ - Hosts -> Compliance -> Tailoring files
+
+__Create Tailoring__ - Upload your Tailoring file xml
+
+### 4.5.3 Assigning a Tailoring file to a Policy
+
+Go to Hosts -> Compliance -> Policies
+
+Select a Policy to edit
+
+Go to 'SCAP Content' tab
+
+Select a Tailoring file from a dropdown and then a profile that comes with it
+![Tailoring a Policy]({{page.images}}/policy_tailoring.png)
+
+## 4.6 Setting up hosts for scans
+
+There are 3 important things that your host needs to have so that it can be scanned properly:
+
+* Policy - you can assign it directly or via hostgroup. See section [4.3]({{site.baseurl}}/plugins/foreman_openscap/{{page.version_short}}/index.html#4.3Assigningpolicytohost) for details.
+
+* foreman_scap_client Puppet module or Ansible role - it will take care of configuring the host with foreman_scap_client. Note that while puppet runs are scheduled periodically, you need to apply Ansible roles to propagate changes as Ansible does not run automatically. foreman_scap_client is a ruby script that runs the openscap scanner with configured options. You can assign module/role to the host such as you would any other module. See the section for [Puppet Classes]({{site.baseurl}}/manuals/{{page.foreman_version}}/index.html#4.2.2Classes) or [Ansible roles]({{site.baseurl}}/plugins/foreman_ansible/2.x/index.html#4.1.1ImportingRoles) for details.
```

Comment:
ansible version again - maybe use another page variable to make sure we link the correct version everywhere?